### ISSUE TITLE
Conda package broken on macOS with python 3.6.3

### ISSUE THREAD

**Issue by bdice** [glotzerlab/freud#154]
**[Original report](https://bitbucket.org/glotzer/freud/issue/154) by Malcolm Ramsay (Bitbucket: [malramsay64](https://bitbucket.org/malramsay64), GitHub: [malramsay64](https://github.com/malramsay64)).**

----------------------------------------

When installing the freud conda package into a new conda environment with python 3.6.3 which is the default python 3.6 version,

```
#!bash
$  conda create -n freud -c glotzer freud
```

the install is fine, but when trying to import freud

```
#!bash
$ python -c "import freud"
Fatal Python error: PyThreadState_Get: no current thread

zsh: abort           python -c "import freud"
```

Unusually by installing python 3.6.1

```
#!bash
$ conda install python=3.6.1
```

the command which previously failed

```
#!bash
$ python -c "import freud"
```

now works fine.

I have also checked using python 3.5, and the 3.5.4 doesn't work, but 3.5.2 does.

I raised an issue in conda-build related to this in that it is not possible to pin the patch version of python as a run or test dependency [https://github.com/conda/conda-build/issues/2571](https://github.com/conda/conda-build/issues/2571).

I have attached the environment.yml file of the failing environment.

**Comment by bdice** [comment]
**Original comment by Bradley Dice (Bitbucket: [bdice](https://bitbucket.org/bdice), GitHub: [bdice](https://github.com/bdice)).**

----------------------------------------

Thanks for reporting this, @malramsay64. We have been working on this issue today. It seems to be caused by library linking to Python that may be avoidable. @joaander is working on a solution. I will update this if/when we release a new conda package that solves the problem.

Related issue #145.

**Comment by bdice** [comment]
**Original comment by Joshua Anderson (Bitbucket: [joaander](https://bitbucket.org/joaander), GitHub: [joaander](https://github.com/joaander)).**

----------------------------------------

Don't link against the python library

This should fix ABI compatibility issues with freud binary builds and different python minor versions.

refs #154

**Comment by bdice** [comment]
**Original comment by Joshua Anderson (Bitbucket: [joaander](https://bitbucket.org/joaander), GitHub: [joaander](https://github.com/joaander)).**

----------------------------------------

Don't link against the python library

This should fix ABI compatibility issues with freud binary builds and different python minor versions.

refs #154

**Comment by bdice** [comment]
**Original comment by Bradley Dice (Bitbucket: [bdice](https://bitbucket.org/bdice), GitHub: [bdice](https://github.com/bdice)).**

----------------------------------------

This problem should be solved with the release of v0.6.3. Please upgrade to the latest freud release if this problem occurs.
